exercises/4/README.md

```diff
@@ -13,35 +13,33 @@ Extend exercise [#1](../1), [#2](../2) and [#3](../3) with the new [objectives](
 
 ## Instructions
 
-1. Rebase your `main` branch on top of `homework/main`.
```

Comment:
I think this should not appear here when you rebase on the homework-repo but im not sure...

Reply:
Yes!